assets/dev/scss/frontend/_container.scss

```diff
@@ -255,3 +258,19 @@
 		--flex-wrap: wrap;
 	}
 }
+
```

Comment:
@TzviRabinovitch I have moved this BC code to the bottom of the file in a clearly marked section so that it will be easier to remove it all at once.

I have asked Doron when we can remove this BC logic, but I haven't received a clear answer yet.

Reply:
@TzviRabinovitch Doron has confirmed that we can delete this in version 3.20.